priv/www/js/tmpl/queue.ejs

```diff
@@ -135,9 +135,11 @@
         <th class="horizontal">Total</th>
         <th class="horizontal">Ready</th>
         <th class="horizontal">Unacked</th>
+        <% if (! queue.hasOwnProperty('members')) { %>
```

Comment:
Do we think there may be more places where conditionals based on queue type are needed? If so, a couple utility functions could be created ... `is_quorum(queue)` and `is_classic(queue)`. If you think that's worthwhile :smile_cat:

Reply:
👍